src/backends/sentry_backend_crashpad.cpp

```diff
@@ -134,8 +140,8 @@ sentry__crashpad_backend_startup(sentry_backend_t *backend)
     sentry__crashpad_backend_user_consent_changed(backend);
 
     crashpad::CrashpadClient client;
-    char *minidump_url = sentry__dsn_get_minidump_url(&options->dsn);
-    std::string url(minidump_url);
+    auto minidump_url = sentry__dsn_get_minidump_url(&options->dsn);
```

Comment:
Let's not use `auto` please, since this makes it harder to read the code and miss mistakes (e.g. `auto` vs `auto *`).